pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "rioxarray >= 0.14.1",
     "rasterio >= 1.3.7",
     "xarray >= 2023.5.0",
+    "spyndex == 0.5.0",
```

Comment:
Maybe it should be equal or greater than 0.5.0 so that nothing breaks if they make future updates?

Reply:
Remaining things to do:

- Add list of accepted band names to error message in read_and _stack_tifs function
- Find workaround to G/G1 naming issues from ```common name = 'green'```

Reply:
Looks like there's a shared common name between the Red Edges, too. So I'll have to think of a way to ask users to distinguish between bands if there are shared common names.

Thanks of the comments!

Reply:
@KavlinClein made the changes re: shared common names. I _think_ we should leave the spyndex as hard 0.5.0, at least for now because we now have manual interventions (the common names fix) that are specific to that version of the package. There's a chance that a new version addresses these issues but with a different solution than we've anticipated. Let me know what you think.